Sources/SpeziAccount/AccountSetup.swift

```diff
@@ -122,8 +122,9 @@ public struct AccountSetup<Header: View, Continue: View>: View {
                     .frame(maxWidth: .infinity)
             }
         }
-            .onChange(of: account.signedIn) {
+            .onChange(of: [account.signedIn, account.details?.isAnonymous]) {
```

Comment:
Seem like we need to adjust the `AccountSetupTests.testSignupWithAnonymousAccount` accordingly?

Reply:
Yes the test was accidentally adjusted to test for the faulty behavior.